tests/cases/fourslash/completionListForUnicodeEscapeName.ts

```diff
@@ -6,7 +6,7 @@
 /////*3*/
 
 verify.completions(
-    { marker: "0", includes: ["B", "\u0042"] },

```

Comment:
Why is unicode string not offered any more.. wouldnt user expect it to be unicode name instead of B ? I dont know what the preference is but old behavior seemed more inclusive of both preferences

Reply:
I don’t think this was ever testing anything. If I understand correctly, this is just running `includes: ["B"]` twice. The unicode escape isn’t observable to `verify.completions` at all.

<img width="159" alt="image" src="https://user-images.githubusercontent.com/3277153/109217813-2a4fa200-776b-11eb-8a37-376612814f79.png">

I removed it in order to change the semantics of `includes` such that each element in the expected array “uses up” an actual completion. We could instead modify the test to run `verify.completions` at marker `0` twice, but again, it’s totally useless.

Reply:
oh i see.. i thought it was "exact" and we showed both options.